mmhtbl.c

```diff
@@ -0,0 +1,184 @@
+/*****************************************************************************/
+/*        Copyright (C) 2017  Thierry Arnoux                                 */
+/*            License terms:  GNU General Public License                     */
+/*****************************************************************************/
+/*34567890123456 (79-character line to adjust editor window) 2345678901234567*/
+
+/* This module manages hash tables. */
+
+#include <stdlib.h>
+#include "mmhtbl.h"
+#include "mminou.h"
+
+/* Static buffer for the linked lists */
+#define NB_LINKEDITEMS 50000L
+#define NO_LINKEDITEM -1
+linked *linkedItems;
+int free_linkedItem;
+flag htinit_done = 0;

```

Comment:
make this a static variable within htinit().  It is completely private implementation detail in this function.